SchoolApplication/Sxhool.WebMvc/Views/Student - Copy/AddStudent.cshtml

```diff
@@ -0,0 +1,122 @@
+﻿@model List<School.Models.Course>
+   
+<form>
+    <h1 class="text-left">Add Student</h1>
+    <div class="row" id="parent" name="myForm">
+
+        <div class="col-md-6">
+
+            <div class="form-group ">
+                <label><b>Name</b></label>
+                <input id="Name" type="text" name="Name" class="form-control" />
+                <label id="Namelb"></label>
+            </div>
+
+            <div class="form-group">
+
+                <label><b>Father Name</b></label>
+
+                <input id="FName" type="text" name="FName" class="form-control" />
+                <label id="FNamelb"></label>
+            </div>
+
+            <div class="form-group">
+
+                <label><b>Email</b></label>
+
+                <input id="email" type="text" name="email" class="form-control" />
+                <label id="emaillb"></label>
+            </div>
+            <div class="form-group">
+
+                <label><b>Date of Birth</b></label>
+
+                <input id="DOB" type="date" value="2014-02-09" name="DOB" class="form-control" />
+
+            </div>
+
+        </div>
+        <div class="col-md-6">
+            <div class="form-group">
+
+                <label><b>Course</b></label>
+                <select id="courseslct" class="form-control select22 " multiple searchable="Search here..">
+                    @foreach (var course in Model)
+                    {
+                        <option value="@course.CourseId" class="select2 ">@course.Name</option>
+
+                    }
+                </select>
+            </div>
+            <div class="form-group">
+
+                <label><b>Phone</b></label>
+                <input id="phone" type="text" name="phone" class="form-control" />
+                <label id="phonelb"></label>
+            </div>
+
+            <div class="form-group">
+                <label><b>Password</b></label>
+
+                <input id="Password" type="password" class="form-control" />
+                <label id="Passwordlb"></label>
+
+            </div>
+            <div class="form-group">
+
+
+                <label><b>Confrim Password</b></label>
+
+                <input id="ConfrimPassword" type="password" class="form-control" />
+                <label id="ConfrimPasswordlb"></label>
+
+            </div>
+            <div class="form-group">
+
+                <button type="button" id="saveBtn" class="form-control btn btn-primary">Save</button>
+                <a href="@Url.Action("Index","Student")" type="button" class="btn btn-info ">Back to List</a>
+
+            </div>
+        </div>
+
+    </div>
+
+
+</form>
+
+
+
+<script>
+$(document).ready(function(){
+    $('#courseslct').select2({
+     placeholder:"choose Courses"});
+});
+
+$('#saveBtn').click(function(){
```

Comment:
Try to use this ajax like a generic function used before or move it to external js file